src/aws-cpp-sdk-core/include/aws/core/utils/crypto/Hash.h

```diff
@@ -9,6 +9,7 @@
 #include <aws/core/utils/crypto/HashResult.h>
 #include <aws/core/utils/memory/stl/AWSStreamFwd.h>
 #include <aws/core/utils/memory/stl/AWSString.h>
+#include <aws/core/utils/HashingUtils.h>
```

Comment:
why are you adding a header to a header while nothing is changed here?

Reply:
because i removed code and forgot to remove header